src/encord_active/app/views/landing_page.py

```diff
@@ -24,6 +24,8 @@ def landing_page():
     with right_col:
         st.markdown((Path(__file__).parent.parent / "static" / "landing-page-documentation.md").read_text())
 
+    load_merged_df()
```

Comment:
Is this necessary? Is it for caching purposes? Perhaps we should to it on a thread then?

Reply:
This loads the datafame to the state with no computations if the DB exists.